apps/studio/components/interfaces/Auth/Users/UserPanel.tsx

```diff
@@ -76,8 +93,26 @@ export const UserPanel = ({ selectedUser, onClose }: UserPanelProps) => {
             value="raw"
             className={cn('mt-0 flex-grow min-h-0 overflow-y-auto', PANEL_PADDING)}
           >
+            <div className="flex items-center mb-2">
+              <Input_Shadcn_
+                autoFocus
+                type="text"
+                placeholder="Search..."
```

Comment:
This is more of a filter don't you think?
```suggestion
                placeholder="Filter..."
```